tests/console/textinfo.pm

```diff
@@ -18,7 +18,14 @@ use testapi;
 # have various useful general info included in videos
 sub run {
     select_console 'root-console';
-    assert_script_run("/home/$username/data/textinfo 2>&1 | tee /tmp/info.txt");
+    # If we're doing this test as the user root, we will not find the textinfo script
+    # in /home/root but rather in /root
+    if ($username eq 'root') {
+        assert_script_run("/root/data/textinfo 2>&1 | tee /tmp/info.txt");
```

Comment:
I would prefer if you set a my $home in here and use that to avoid duplicating 95%

Reply:
Will do.